doc/articles/howto-consume-webservices.md

```diff
@@ -633,15 +616,21 @@ In this sample application you will be adopting the Model-View-ViewModel (MVVM)
         }
         else
         {
-            await Dispatcher.RunAsync(CoreDispatcherPriority.Normal, callback);
+            var completion = new TaskCompletionSource();
+            Dispatcher.TryEnqueue(() =>
+            {
+                callback();
+                completion.SetResult();
+            });
+            await completion.Task; 
         }
     }
     ```
 
-    > [!NOTE]
-    > As **WASM** is currently single-threaded, the **Uno.WASM** implementation of `Dispatcher.HasThreadAccess` always returns `false` by default. This default has been chosen to prevent live-locking for certain frameworks that expect a multi-threaded environment. This app overrides that default WASM behavior here via conditional compilation and always invokes the callback directly on WASM.
+> [!NOTE]
+> As **WASM** is currently single-threaded, the **Uno.WASM** implementation of `DispatcherQueue.HasThreadAccess` always returns `false` by default. This default has been chosen to prevent live-locking for certain frameworks that expect a multi-threaded environment. This app overrides that default WASM behavior here via conditional compilation and always invokes the callback directly on WASM.
```

Comment:
@jeromelaban I'm not sure if this is still true or if it is outdated statement. Can you confirm?

Reply:
This is still true. There's no direct API to determine if threading is supported (something that .NET 8 will provide).